### ISSUE TITLE
Add accessibility section

### ISSUE THREAD

**Issue by LJWatson** [w3c/input-events#110]
In the [WebApps charter](https://www.w3.org/2019/05/webapps-charter.html#scope) it says:
>Where there are implications for implementors, developers, or users, in the areas of accessibility, internationalization, privacy, and security, each specification must have a section that describes relevant benefits, limitations, and best practice solutions for that particular area.>

To this end each spec needs to have an accessibility section. For the avoidance of doubt in the reader, the section should be included even if it indicates there are no known accessibility considerations.

Relates to w3c/webappswg#15

**Comment by johanneswilm** [comment]
Given comments made by APA WG on no accessibility-linked changes required from us here [1] and that we are no longer part of the WebApps Wg, I will close this issue. Please reopen or create a new issue if changes are needed anyway.

[1] https://lists.w3.org/Archives/Public/public-apa/2022Jan/0004.html
